packages/core/src/resolvers/object.ts

```diff
@@ -15,6 +17,34 @@ export const resolveObject = ({
   combined?: boolean;
   context: ContextSpecs;
 }): ResolverValue => {
+  if (typeof schema === 'object' && (schema as SchemaObject)?.allOf) {
```

Comment:
Do you think it can be moved in the combineSchemas function? And maybe refactor that one only? It's executed in getters/object.ts and line `41`

Reply:
I tried but nested allOf is not correct.  Because `mix in` is not working at the following line.
https://github.com/anymaniax/orval/blob/master/packages/core/src/getters/object.ts#L93

Reply:
Ok then for schemas we could have a problem no? Since [here](https://github.com/anymaniax/orval/blob/master/packages/core/src/generators/schema-definition.ts) it’s calling resolveValue only

Reply:
It's possible, but I can't get to that line in the OpenAPI description I know. Which will be the sample?

```yaml
schema:
  allOf:
    - $ref: 'schemas/pet.yaml#/components/schemas/Pet'
    - required:
        - name
    - type: object
      properties:
        name:
          description: required name
        owner:
          deprecated: true
```

Reply:
I got it. Case by components top level allOf. I will add test case.

Reply:
Converted to getters/object.ts. The components side also worked correctly.